lib/jsonapi/include_directive.rb

```diff
@@ -68,5 +71,20 @@ def to_string
 
       string_array.join(',')
     end
+
+    class InvalidKey < StandardError; end
+
+    private
+
+    def valid?(key)
+      key.match(valid_json_key_name_regex)
+    end
+
+    def valid_json_key_name_regex
+      # not start with hyphen/underscore/space
+      # not end with hyphen/underscore/space
+      # contains a-zA-Z, 0-9, and hyphen/underscore/space in member names
```

Comment:
Could you replace these comments with a link to http://jsonapi.org/format/#document-member-names? Also, the regex still seems to exclude some allowed characters such as 
> U+0080 and above (non-ASCII Unicode characters; not recommended, not URL safe)